.github/workflows/release.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           DOCKER_REPO: ${{ secrets.DOCKER_REPO }}
         run: |
           docker buildx build \
-            --platform linux/amd64,linux/arm64,linux/arm/v7 \
+            --platform linux/amd64 \
```

Comment:
Why remove arm? for the testing?

Reply:
The release build failed on arm64 without returning a clear error message. I didn't want to spend a lot of time debugging that, so I just removed arm.
I will revert the change eventually.

Reply:
This has been removed from the branch.